src/app/public/modules/split-view/split-view.component.scss

```diff
@@ -0,0 +1,25 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
```

Comment:
Same question about the compat being included

Reply:
See answer above.